src/jsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import { VNode } from 'preact';
```

Comment:
For any definition files other than the one referenced in `package.json` TypeScript expects them to be next to the real code. In our case `jsx.js` is referenced from a file of the same name in the root of this repository. TypeScript is unable to understand that indirection. Moving `jsx.d.ts` to the repo root will fix this 👍

Reply:
You're right. Give me a sec...